src/sincosint.jl

```diff
@@ -233,12 +233,13 @@ Compute the sine integral function of ``x``, defined by
 
 ```math
 \operatorname{Si}(x)
-:= \int_0^x \frac{\sin t}{t} \, \mathrm{d}t
+:= \int_0^x \frac{\sin t}{t} \, dt
```

Comment:
Why? `\mathrm{d}t` looks better to me.

Reply:
This is just for consistency with some other integral formulas which don't use `\mathrm`.

Reply:
Then why not changing the other integral formulas which don't use `\mathrm`?

Reply:
Fair point. It was really a swing either way over which form to use, and it makes just as much sense to do the `\mathrm`. Will be pushing that soon. Thank you!